android/app/src/main/python/electroncash_gui/android/strings.py

```diff
@@ -44,8 +45,12 @@ def _(s):
 _("Request")
 _("Restore from seed")
 _("Show seed")
+_("Sign unbroadcasted transaction")
 _("Size")
+_("The signed transaction has been")
```

Comment:
Sentence is incomplete.

Reply:
I did this because the copytoclipboard function wanted something to precede "copied to Clipboard". I guess "Signed Transaction copied to clipboard" works just as well, so I made it that.

Reply:
Oh yes, I'd forgotten about that. Yes, it's better to just use a simple noun in that context, otherwise it could be difficult to translate into other languages.